### DISCUSSION TITLE
How to download synced computers from google drive?

### DISCUSSION THREAD

**Discussion by karnathe** [iterate-ch/cyberduck#16408] [Q&A]
Hello, I checked the FAQ and did not see this, I am very sorry if this has been answered. I am using cyberduck to download my entire drive from google drive, it is working perfectly. 

However, I do not know **how to download my synced computers**. Please see the attached image with my lovingly named computers.

![image](https://github.com/user-attachments/assets/1255f42b-36f8-44f1-ac40-102a31879a4d)

In cyberduck GUI i see my drive, shared drives, etc, but I do not see the "computers" section.

Thank you very much for your help!

**Comment by tatze96** [comment]
Mounting and accessing the _Computers_ section of Google Drive is not supported.
